WooCommerce/WooCommerceUITests/Tests/UniversalLinksTests.swift

```diff
@@ -0,0 +1,31 @@
+import UITestsFoundation
+import XCTest
+
+final class UniversalLinksTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+
+        let app = XCUIApplication()
+        app.launchArguments = ["logout-at-launch", "disable-animations", "mocked-wpcom-api", "-ui_testing"]
+        app.launch()
+    }
+
+    func test_load_payments_universal_link() throws {
+        try LoginFlow.login()
+
+        try ExternalAppScreen().openUniversalLinkFromSafariApp(linkedScreen: "payments")
+        try PaymentsScreen().verifyPaymentsScreenLoaded()
+    }
+
+    func test_load_orders_universal_link() throws {
+        // run test only on iPhone, on iPad there's an issue where the incorrect order is opened
+        if UIDevice.current.userInterfaceIdiom == .phone {
+            try LoginFlow.login()
+            let order = try GetMocks.readSingleOrderData()
+
+            try ExternalAppScreen().openUniversalLinkFromSafariApp(linkedScreen: "orders")
+            try SingleOrderScreen().verifySingleOrder(order: order)
+        }
```

Comment:
also decided to move the tests into a new universalLinksTests file so that when we skip this on iPad it doesn't have to go through the login flow (~20seconds) before skipping the test